gogs_client/entities.py

```diff
@@ -13,17 +13,24 @@ def json_get(parsed_json, key):
     return parsed_json[key]
 
 
-class GogsUser(object):
+class GogsEntity(object):
+    def __init__(self, json):
+        self.json = json
```

Comment:
because all API resources exposed shall store some JSON data, it seemed logical to offer a similar approach. Then improving like ② is possible.